conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ArrayFireConan(ConanFile):
+    name = "arrayfire"
+    version = "3.7.1"
+    license = "BSD"
+    author = "jacobkahn jacobkahn1@gmail.com"
+    url = "https://github.com/arrayfire/arrayfire"
+    requires = []
+    description = "ArrayFire: a general purpose GPU library"
+    topics = ("arrayfire", "gpu", "cuda", "opencl", "gpgpu",
+              "hpc", "performance", "scientific-computing")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "backend": ["CPU", "CUDA", "OPENCL"]}
```

Comment:
Shouldn't unified backend also be present ?

Reply:
Done. Also changed so users can opt-in to multiple backends.

Reply:
added 👍